<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shader Bro</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Asset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HPushButton" name="WorkRefresh">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Reread shaders from disk</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2347.png</normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2347.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="HListWidget" name="Asset"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HListWidget" name="Task"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HComboBox" name="Version"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Shader</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HListWidget" name="Shader"/>
     </item>
     <item>
      <widget class="QPushButton" name="ImportShader">
       <property name="text">
        <string>Import shader</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyToSelection">
       <property name="text">
        <string>Apply shader to selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Work</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Work">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HPushButton" name="WorkLoad">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2233.png</normaloff>P:/global/code/pipeline/bootstrap/psycons/icon_packs/EMOJI/icon.2233.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HListWidget</class>
   <extends>QListWidget</extends>
   <header>hlistwidget.h</header>
   <slots>
    <slot>blah()</slot>
    <slot>redraw()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>HComboBox</class>
   <extends>QComboBox</extends>
   <header>hcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>HPushButton</class>
   <extends>QPushButton</extends>
   <header>hpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
